service/history/api/reapplyevents/api.go

```diff
@@ -161,6 +162,7 @@ func Invoke(
 					ndc.EventsReapplicationResetWorkflowReason,
 					toReapplyEvents,
 					nil,
+					allowResetWithPendingChildren,
```

Comment:
this api is only used by replication stack I believe, so should also be false I think. cc @yux0

Reply:
keeping this consistent with existing behavior.